CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
 - **Onboarding Service Provider**
   - removed deprecated page onboardingServiceProvider and every related link [#1261](https://github.com/eclipse-tractusx/portal-frontend/pull/1261)
 
+### Bugfixed
+
+- **Application Request**
+  - fixed endurl issue in the retriggering of clearing house step [#1268] (https://github.com/eclipse-tractusx/portal-frontend/pull/1268)
```

Comment:
Please also put it under Change, according to the PR title and description I updated yesterday.

Reply:
updated